lib/swaggerValidator/trafficValidator.ts

```diff
@@ -151,8 +156,25 @@ export class TrafficValidator {
           if (!this.trafficOperation.get(swaggerFile)?.includes(opInfo.info.operationId)) {
             this.trafficOperation.get(swaggerFile)?.push(opInfo.info.operationId);
           }
+          if (
+            validationResult.requestValidationResult.isSuccessful === false ||
+            validationResult.requestValidationResult.isSuccessful === undefined ||
+            validationResult.responseValidationResult.isSuccessful === false ||
+            validationResult.responseValidationResult.isSuccessful === undefined ||
+            validationResult.runtimeException !== undefined
+          ) {
+            if (this.validationFailOperations.get(swaggerFile) === undefined) {
+              this.validationFailOperations.set(swaggerFile, []);
```

Comment:
Can you push operationId in this set and change followed if to else if?

Reply:
operaionId is pushed to this set already, do you mean to get all these operations together with the coverage result?

Reply:
here we should use if but not else if: after the set is initiated, we need to push failed operationid.